proto/process/agent.proto

```diff
@@ -1576,6 +1576,46 @@ message DataStreamsAggregations {
 	repeated KafkaAggregation kafkaAggregations = 3;
 }
 
+// A list of the supported operations for the Postgres database.
+// Other operations will not be monitored and will not be included in the agent's output.
+enum PostgresOperation {
+	PostgresUnknownOp = 0;
+	PostgresSelectOp = 1;
+	PostgresInsertOp = 2;
+	PostgresUpdateOp = 3;
+	PostgresDeleteOp = 4;
+	PostgresAlterOp = 5;
+	PostgresCreateOp = 6;
+	PostgresDropOp = 7;
+}
+
+// Represents the statistics for a single Postgres operation.
+message PostgresStats {
+	// The name of the table that this PostgresStats represents.
+	string table_name = 1;
+	// The operation that this PostgresStats represents.
+	PostgresOperation operation = 2;
+	// A protobuf encoded sketch of all the latencies for this operation. In nanoseconds.
+	bytes latencies = 3;
```

Comment:
Don't we have a better way to describe the latencies then bytes type?

Reply:
no, the protobuf representation is less effective (memory and cpu) than using raw bytes
we're sticking to the http/http2 implementation here